bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/input.js

```diff
@@ -1,8 +1,14 @@
 import { pi, pb, pt, pd } from '../helpers.js'
 
+export const ptType = pt('type', 'HTML Input Type',
+  'HTML input type for different OpenHAB Item types.\
```

Comment:
I do think the previous description shouldn't be removed completely, as it provided helpful guidance if you want to set the type manually. The assumed default types should be combined with the existing description.

Reply:
Just understood what you did here, that's better than simply linking to the docs!